tests/k8s_test.go

```diff
@@ -149,6 +149,10 @@ func TestK8sPodNode(t *testing.T) {
 	testNodeCreationFromConfig(t, "pod", podName)
 }
 
+func TestK8sReplicationControllerNode(t *testing.T) {
+	testNodeCreationFromConfig(t, "replicationcontroller", "nginx")
```

Comment:
the initial test scenario I have  done for each new resource is minimalist (only checking for creation of the graph node) - next planned are full scenarios (checking multiple graph nodes and graph edges) - so when testing full scenarios it will also include a storage scenario involving for example a ReplicationController (it maintains a fixed number of running Pods in the cluster) and Pods (among other resources).

Beyond full (static) scenarios I will next move to dynamic scenarios in which elements are added/deleted/updated.